coc.html

```diff
@@ -1,327 +1,288 @@
+			Code of Conduct is based on:
+		<ul>
+			<li><a target="_blank" rel="noreferrer"
+					href="https://www.contributor-covenant.org/version/2/1/code_of_conduct">Contributor Covenant
+					(v2.1)</a>
+				and has been modified as allowed by the
+				<a target="_blank" rel="noreferrer" href="https://choosealicense.com/licenses/cc-by-4.0">CC-BY-4.0</a>
+				license.
+			</li>
+			<li>
+				<a target="_blank" rel="noreferrer" href="https://www.recurse.com/social-rules">Recurse Center's Social
+					Rules</a>.
+			</li>
+		</ul>
+		</p>
+
+		<h2 id="our-pledge">Our Pledge</h2>
+		<p>
+			We as members, contributors, and leaders pledge to make
+			participation in our community a harassment-free experience for
+			everyone, regardless of age, body size, visible or invisible
+			disability, ethnicity, sex characteristics, gender identity and
+			expression, level of experience, education, socio-economic
+			status, nationality, personal appearance, race, caste, color,
+			religion, or sexual identity and orientation.
+		</p>
+		<p>
+			We pledge to act and interact in ways that contribute to an
+			open, welcoming, diverse, inclusive, and healthy community.
+		</p>
+
+		<h2 id="our-standards">Our Standards</h2>
+		<p>Examples of <strong>acceptable behavior</strong> include:</p>
+		<ul>
+			<li>Demonstrating empathy and kindness toward other people.</li>
+			<li>Accommodating for people with disabilities.</li>
+			<li>Abide by the rules of the community and event.</li>
+			<li>
+				Being respectful of differing opinions, viewpoints and
+				experiences.
+			</li>
+			<li>Giving and gracefully accepting constructive feedback.</li>
+			<li>
+				Accepting responsibility and apologizing to those affected
+				by our mistakes and learning from the experience.
+			</li>
+			<li>
+				Focusing on what is best not just for us as individuals, but
+				for the overall community.
+			</li>
+		</ul>
+		<p>Examples of <strong>unacceptable behavior</strong> include:</p>
+		<ul>
+			<li>Not co-operating with the organising team.</li>
+			<li>
+				The use of sexualized language or imagery, and sexual
+				attention or advances of any kind.
+			</li>
+			<li>Repeatedly flouting event rules.</li>
+			<li>
+				The use of derogatory or discriminatory words, sentiments or
+				statements against any race, gender, sexuality and/or
+				religion.
+			</li>
+			<li>
+				Trolling, insulting or derogatory comments, and personal or
+				political attacks.
+			</li>
+			<li>Public or private harassment.</li>
+			<li>
+				Publishing others’ private information, such as a physical
+				or email address, without their explicit permission.
+			</li>
+			<li>
+				Other conduct which could reasonably be considered
+				inappropriate in a professional setting.
+			</li>
+		</ul>
+		<br/>
+		<p>In addition to these rules of conduct, everyone participating in events is encouraged to abide by the Recurse Center Social Rules:</p>
```

Comment:
And let's hyperlink this as well.

```html
<a target="_blank" rel="noreferrer" href="https://www.recurse.com/social-rules">Recurse Center's Social Rules</a>.
```

Reply:
Pushed a new commit that should take care of this. Resolving conversation. Thanks!